corehq/tabs/tabclasses.py

```diff
@@ -644,11 +644,14 @@ def sidebar_items(self):
                 request=self._request, domain=self.domain))
 
         from corehq.apps.users.models import HqPermissions
-        has_view_data_dict_permission = self.couch_user.has_permission(
+        has_edit_or_view_data_dict_permission = self.couch_user.has_permission(
```

Comment:
Shouldn't the `.normalize()` method make checking view permission sufficient?

Reply:
Ya as it stands, having edit permission will also enable view permission. I wanted to be more explicit and not have edit coupled to view but rethinking it now, it's unnecessary. Changed in [dad0b29](https://github.com/dimagi/commcare-hq/pull/32345/commits/dad0b2981d32e10b91c8f1ce2a50429a9e68708f)